api/specs/webserver/openapi-auth.yaml

```diff
@@ -21,6 +21,48 @@ paths:
         default:
           $ref: "#/components/responses/DefaultErrorResponse"
 
+  /auth/verify-phone-number:
```

Comment:
Should we maybe consider to rate limit this route? I think a rate limit by IP should be sufficient.

Reply:
yes, but I would like to do it with traefik
I wait to next iteration where we add the front-end. There we might have to change the paths again